dex/src/test/scala/com/wavesplatform/dex/model/OrderBookSpec.scala

```diff
@@ -322,6 +354,12 @@ ${canceledOrders.mkString("\n")}
     }
   }
 
+  private def obInvariant(ob: OrderBook): Unit = {
+    val firstAskPrice = ob.asks.headOption.map(_._1).getOrElse(Long.MaxValue)
+    val firstBidPrice = ob.bids.headOption.map(_._1).getOrElse(Long.MinValue)
+    firstAskPrice should be > firstBidPrice
+  }
+
```

Comment:
Bad name imho
What about `assertSpreadSign`?

Reply:
An invariant is a property that should be always held. I see here only two properties:
1. Asks and bids have right order;
2. Asks and bids don't intersect each other.

The first is guaranteed by TreeMap. so there is only one invariant. Changed to `obAskBidIntersectionInvariant`